bot.py

```diff
@@ -451,7 +451,8 @@ async def attempt_disconnect(self):
         if self.voice_client.is_playing():
             return
 
-        if self.last_played_time < self.DISCONNECT_TIMER_SECONDS:
+        time_since_last_played = time.time() - self.last_played_time
+        if time_since_last_played < self.DISCONNECT_TIMER_SECONDS:
```

Comment:
```suggestion
        if time.time() - self.last_played_time < self.DISCONNECT_TIMER_SECONDS:
```

Reply:
Oneliner :hot_face: 

Slightly less readable, but doesn' make a new "unnecessary" variable.

Reply:
Yeah I guess this is clear enough